eyeshade/controllers/owners.js

```diff
@@ -187,8 +188,10 @@ v3.bulk = {
           entry = await publishers.findOne({ publisher: channelId })
           if (!entry) continue
 
-          await publishers.update({ publisher: channelId }, { $set: { owner: owner.owner_identifier } },
-                                  { upsert: true })
+          if ((entry.owner) && (cleanup.indexOf(entry.owner))) cleanup.push(entry.owner)
+          await publishers.update({ publisher: channelId }, {
+            $set: { owner: owner.owner_identifier, authority: owner.owner_identifier }
```

Comment:
why do we set two properties both to the same value?

Reply:
in the future, there may be a different entity that authorizes an owner...